app/javascript/components/remove-catalog-item-modal.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Modal } from 'patternfly-react';
+
+const parseApiError = (error) => {
+  const { data: { error: { message } } } = error;
+  return message;
+};
+
+const removeCatalogItems = (catalogItems) => {
+  let apiPromises = [];
+
+  miqSparkleOn();
+  catalogItems.forEach(item => {
+    apiPromises.push(API.post(`/api/service_templates/${item.id}`, {action: 'delete'}, {skipErrors: [400, 500]})
```

Comment:
Does the API ever return 500 in situations where you would not want the error modal?

Reply:
Please see https://github.com/ManageIQ/manageiq-ui-classic/pull/5139#issuecomment-466949652

Reply:
@rvsia that's about 400, Bad Request.

400 is fine, that means the user did something wrong.
500 means the server is broken.